app/views/tcsp/summary.scala.html

```diff
@@ -87,8 +98,17 @@
             ) {
                 <ul class="list list--comma">
                     @model.providedServices.map { a =>
-                        @a.services.map { x =>
-                            <li>@x.getMessage</li>
+                        @if(a.services.collectFirst {case Other(_) => true}){
```

Comment:
Do we need this if/else? I think that just having lines 102-107 might achieve the same outcome?

Reply:
this was to ensure that other will appear as last if there are some other provided services, now I'm not sure what was in lines 102-107 as everything moved but if you meant that was was in the body of if part only, I think you are right. This may be sorted as well in controller as @samzaroughmrc suggested

Reply:
done